README.md

```diff
@@ -39,11 +39,46 @@ This command starts a local development server and opens up a browser window. Mo
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
-## Localization
+## How to translate
 
 Translations are not pushed to the repo and prepared on-the-fly (`i18n` folder is added to `.gitignore`).
 
 Here's how you can debug translations locally.
 
 1. Download translations: `CROWDIN_PERSONAL_TOKEN="YOURTOKEN" yarn run crowdin download`
 2. Run Docusaurus with the language of your choice: `yarn run start -- --locale de`
+
+## How to generate DNS stamps
```

Comment:
You should probably add an explanation that DNS stamps are needed when you edit the Known DNS providers article.